src/harbour-tuschbox.cpp

```diff
@@ -53,12 +53,16 @@ int main(int argc, char *argv[])
 
     qmlRegisterType<QQuickFolderListModel>("harbour.tuschbox.Tuschbox", 1, 0, "FolderListModel");
 
+
     QQuickView *view = SailfishApp::createView(); // I get a white background with this.
     view->setSource(SailfishApp::pathTo("qml/harbour-tuschbox.qml"));  // So I do this ;)
 
     FM *fileAction = new FM();
     view->engine()->rootContext()->setContextProperty("_fm", fileAction);
 
+    Settings *settings = new Settings();
```

Comment:
Cool, and totally agree. 
I would like to do some additional improvements like "remember to the last used board", and user browseable icons, I am also thinking about creating some way to being able to create soundboard "packages" by the users.